_includes/build/bcda_v2.html

```diff
@@ -19,13 +19,15 @@ <h2>
             </tr>
             <tr>
                 <td>
-                    <code>curl -X GET 'https://api.bcda.cms.gov/api/v1/Patient/$export' \ 
-                        -H "accept: application/fhir+json" \ -H "Prefer: respond-async" \ 
+                    <code>curl -X GET 'https://api.bcda.cms.gov/api/v1/Patient/$export' \
```

Comment:
When I copy this one, there is whitespace at the end after the `\`. It's hard to tell here while highlighting the text; do you see that as well?